pacman/README.md

```diff
@@ -0,0 +1,155 @@
+
+# Scoring
+
+### Search (1/3)
+
+The score function is computed as follows:
+
+`score = -#time steps + 10*#number of food dots eaten + (500 when finished)`
+
+# Instructions
+
+## Introduction 
+
+![Pacman maze example](https://github.com/glouppe/info8006-introduction-to-ai/blob/pacman_project/pacman/pacman_game.png "Pacman maze")
+
+In this classical Pacman game, the player navigates Pacman through a maze filled of food dots. The goal is to collect them all in a minimum amount of time (Part 1/3). Ghosts, which are Pacman's enemies, may also be part of the game. In this configuration, Pacman needs to avoid them while collecting all the food dots (Part 2/3). To help him, Pacman may have access to capsules in the maze, giving him the ability to beat Ghosts, resetting them in their initial positions, in a short amount of time. The game ends if either Pacman have collected all the dots (winning end) or Pacman collides with one of the Ghosts (losing end). Cherry on the cake, some walls on the game may have a particular characteristic. Indeed, they can randomly be crossed or not over time (Part 3/3).
+
+The project is splitted into three parts. Your task is to design a family of searching agents for each part of the project (details are provided in the three next sections below). 
+
+## Searches (Part 1/3)
+
+Before the implementation, you need to formalize the Pacman game as a search problem, as seen in [Lecture 2](https://glouppe.github.io/info8006-introduction-to-ai/?p=lecture2.md). Then, implement the following searching algorithms :
+
+ - Depth-First Search (DFS)
+ - Breadth-First Search (BFS)
+ - Uniform Cost Search (UCS)
+ - A* with *your own heuristic*
+
+You need to justify the choice of the heuristic for A* regarding the layout of the map, including the distribution of the food dots, in terms of final score/total computation time.
+
+
+
+### Deliverable 1/2 - Implementation
+
+You need to send, at least, a Python 3 source code file, containing the [class template](https://github.com/glouppe/info8006-introduction-to-ai/tree/master/pacman/pacmanagent.py) completed on your own. Include also the other dependencies you have implemented inside a subfolder. Implementation [examples](https://github.com/glouppe/info8006-introduction-to-ai/tree/master/pacman/) are provided. Your source code needs to be formatted in order to follow the [PEP-8](https://www.python.org/dev/peps/pep-0008/) style conventions. Make sure to properly follow the instructions contained in the class template. 
+
+### Deliverable 2/2 - Report
+
+You need to provide a report in PDF format of 4 pages max (not including appendices for results). Your report must (i) contains the formalization of the Part 1 as a search problem, (ii) describes and justify carefully the (admissible) heuristics you have considered, (iii) show and discuss benchmark results, comparing the performance of your agent with the other algorithms you have implemented.
+
```

Comment:
part (iii) should be more precise. We must specify exactly the experiments we expect them to do.